datasets/gb/coh/gb_coh_disqualified.yml

```diff
@@ -118,3 +117,8 @@ lookups:
         value: "2031-07-16"
       - match: "9999-12-31"
         value: null
+  type.name:
+    options:
+      # as per https://find-and-update.company-information.service.gov.uk/disqualified-officers/natural/nPada61zCehhR2wnbTffM3PbVXA
+      - match: '5191/2015'
+        value: GSE LONDON LIMITED
```

Comment:
so is the name of the company `5191/2015` in the gb_coh data? and we're inferring that the company's name is `GSE LONDON LIMITED`?

when I search for the company I don't find it, so It isn't clear what we're mapping to this name
https://find-and-update.company-information.service.gov.uk/search/companies?q=5191